tools/automate_bindings.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/python
+
+""" Generate bindings of given Wt's class and all its dependencies.
+
+Dependencies are all other classes from interface of methods
+of given class (types of arguments and return values).
+"""
+
+import argparse
+import os
+
+from pygccxml import declarations
+from pygccxml import parser
+from pygccxml import utils
```

Comment:
Can you `import pygccxml` and use full names (e.g. `pygccxml.parser`)?

Name `parser` is reused in `main` function for arguments parser.